### ISSUE TITLE
[Bug]:  Stretchly cannot skip the break when mac do not disturb is on

### ISSUE THREAD

**Issue by stevechoris** [hovancik/stretchly#1172]
### Version

- [X] I'm using version 1.10.0

### Known issues

- [X] I've checked Known issues

### Existing issues

- [X] I've checked Existing issues

### What operating system are you using?

macOS

### Operating System Version

macOS Monterey 12.4

### Reproduction steps

1. open mac do not disturb
2. 
3.
...
**Reproduces how often:** [What percentage of the time does it reproduce?]

### Expected Behavior

Stretchly stop timer and skip the break

### Actual Behavior

Stretchly still continue timer and show the break screen.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hovancik** [comment]
Hi, I am not sure what exactly you mean in "Expected Behavior", but when DND is on, it is correct behavior to not allow breaks. Stretchly respects DND settings and show no breaks in DND.

**Comment by stevechoris** [comment]
> Hi, I am not sure what exactly you mean in "Expected Behavior", but when DND is on, it is correct behavior to not allow breaks. Stretchly respects DND settings and show no breaks in DND.

In fact,  now if i turn on DND, the break page will still show. My system version is monterey 12.5。

**Comment by hovancik** [comment]
@stevechoris could you please share the logs when that happens?

**Comment by stevechoris** [comment]
ok, where is the log path, please?

**Comment by hovancik** [comment]
https://github.com/hovancik/stretchly#debugging

**Comment by tomasedl** [comment]
I have same issue. My system version in 12.5.1

[Here](https://drive.google.com/file/d/1_0o8Dwo5DEfpahPqa1JKjzGLRZKvESKM/view?usp=sharing) are the logs. It happens for some times.

I took a quick look at the logs and it looks like the DND mode detection is the problem. I tried to turn DND on and no record appears in logs. I have DND set to be turned on working days at 10:00 and that is not in logs as well.
